helm_deploy/values-dev.yaml

```diff
@@ -13,6 +13,7 @@ generic-service:
     API_BASE_URL_OAUTH: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     API_BASE_URL_PRISON: https://api-dev.prison.service.justice.gov.uk
     FEATURE_REVIEWADDEDSYNCMECHANISM: "DOMAIN_EVENT"
+    FEATURE_INCENTIVES_DATA_SOURCE_OF_TRUTH: false
```

Comment:
I'm new to Spring Boot but you may need to add it to `src/resources/(application.yml|application-dev.yml)` (similar to what done [for the other property](https://github.com/ministryofjustice/hmpps-incentives-api/blob/main/src/main/resources/application.yml#L5-L12))

Reply:
Sorry just noticed this is added to the `application.yml` below 👌